artiq/coredevice/comm_analyzer.py

```diff
@@ -38,14 +38,21 @@ def get_analyzer_dump(host, port=1382):
     sock = socket.create_connection((host, port))
     try:
         r = bytes()
+        endian = sock.recv(1)
```

Comment:
Move to ``decode_dump``?
I don't think keeping the dump format compatible between major ARTIQ versions is very important, AFAICT in typical these dump files are used immediately after they are obtained and are not kept around for long.
Just add an entry in the release notes that say there's a new byte at the beginning to specify endianness.